jpsreport/methods/ConfigData_DIJ.h

```diff
@@ -0,0 +1,50 @@
+/**
+ * \file        ConfigData_DIJ.h
+ * \copyright   <2009-2019> Forschungszentrum Jülich GmbH. All rights reserved.
+ *
+ * \section License
+ * This file is part of JuPedSim.
+ *
+ * JuPedSim is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * any later version.
+ *
+ * JuPedSim is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with JuPedSim. If not, see <http://www.gnu.org/licenses/>.
+ *
+ * \section Description
+ * A simple class for storing the configuration data for Method D, I and J
+ *
+ **/
+
+#pragma once
+
+#include <vector>
+
+struct ConfigData_DIJ {
+    //parameters that are provided for measurement areas
+    std::vector<bool> individual_FD_flags;
+    std::vector<int> start_frames;
+    std::vector<int> stop_frames;
+    std::vector<int> areaIDs;
```

Comment:
Is there a mapping from given measurement ID to vector index?

Reply:
Currently, jpsreport reads the measurementIDs for a certain method in this vector and then works on them after each other. There is no real mapping.